### ISSUE TITLE
Repurposed Structures prevents AE2 meteorites from spawning

### ISSUE THREAD

**Issue by FreeFull** [TelepathicGrunt/RepurposedStructures-Fabric#162]
Minecraft version 1.18.0
Mod list:
```
appliedenergistics2-10.0.0-alpha.1.jar
cloth-config-6.0.45-fabric.jar
fabric-api-0.44.0+1.18.jar
repurposed_structures_fabric-3.1.5+1.18.0.jar
```
latest.log: https://gist.github.com/FreeFull/31dd81f0046be1d5ef20e556e70e3567

As far as I can tell, the meteorites are the only structure that fails to spawn with RS present. All the AE2 ores and such do spawn normally.

**Comment by TelepathicGrunt** [comment]
fixed in v3.1.6

**Comment by itsdinkd** [comment]
This also happens on 1.17

can you release the fix for that as well?!

**Comment by TelepathicGrunt** [comment]
Uh. Beside support being dropped for 1.17 and below, the issue doesn’t actually exist in 1.17. The issue in 1.18 was RS’s deep noise copier wasn’t copying a specific configuredstructure field that was added only in 1.18 and so, it was conflicting with Fabric API’s BiomeModification API in that Mc version. As far as I know, RS works with other structure mods just fine in 1.17 and not even AOF 6 or 7 reported issues of structures not spawning. If there was an issue, my own mod would break itself because 1.17 RS and other mods uses BiomeModification API to add structures to biomes. 

I’ll test tomorrow and show pics.

**Comment by itsdinkd** [comment]
I have more mods in my pack than AOF not sure how that's relevant, however,
if it isn't due to RS than it's something else. I figured it was because it
was fixed for 1.18. Never tried debugging to find out the cause.

**Comment by TelepathicGrunt** [comment]
I booted up 1.17.1, put on RS, AE2, and many other structure mods. All seem to be working for me.
![image](https://user-images.githubusercontent.com/40846040/144750734-771787ab-2cee-4d1d-890e-11ec44f854a7.png)

Your best bet is check AE2's config first to make sure the meteors are not turned off. Then start looking for what mod could be conflicting
